libparsec/crates/client/src/workspace/store/file_updater.rs

```diff
@@ -26,7 +26,7 @@ pub(crate) enum ForUpdateFileError {
     #[error("Path doesn't exist")]
     EntryNotFound,
     #[error("Path points to entry that is not a file")]
-    EntryNotAFile,
+    EntryNotAFile { entry_id: VlobID },
```

Comment:
The value `entry_id` is not used in the error message

Reply:
done. (I've also apply this to other errors with similar `entry_id` :+1: )